zeppelin-server/src/main/java/org/apache/zeppelin/rest/ClusterRestApi.java

```diff
@@ -152,8 +160,15 @@ public Response getClusterNodes(){
     return new JsonResponse(Response.Status.OK, "", nodes).build();
   }
 
-  private String formatIntpLink(String intpName) {
-    return String.format("<a href=\"/#/cluster/%s\">%s</a>", intpName, intpName);
+  private String formatDateTime(Object localDateTime) {
```

Comment:
i think the function should be strongly types? one approach is `private String formatDateTime(LocalDateTime localDateTime)`

and then `String dateTime = formatDateTime((LocalDateTime)serverStartTime);`

I think this will be more maintainable.

Reply:
done. and [CI pass](https://travis-ci.org/liuxunorg/zeppelin/builds/559279220)